src/pkm/PK4.ts

```diff
@@ -40,6 +40,7 @@ export class PK4 {
   movePPUps: number[]
   ivs: types.Stats
   isEgg: boolean
+  // eggLocationIndex: number = 0
```

Comment:
Same here. also are you removing this line because it's a duplicate?

Reply:
You know this part is confusing. When running the `Gen##SAV.test.ts` file it gives an error saying `eggLocation` is undefined in PK4, thus I added this line. When building OPENHOME, this line must be commented out otherwise it returns an error. So my answer I guess is that an individual test needs to be created for the PK4 file to make sure that this can be run without `eggLocationIndex`. Until then we can remove it.